tests/conftest.py

```diff
@@ -1,25 +1,19 @@
 import asyncio
 import itertools
-import json
 import logging
 import os
 import random
 import re
-import shutil
-from copy import deepcopy
-from pathlib import Path
-from subprocess import PIPE, STDOUT, CalledProcessError, Popen, run
+from subprocess import PIPE, STDOUT, CalledProcessError, Popen
 
 import pytest
 
-from .node import Node
+from sdk.python import localcluster
+from sdk.python.localcluster.node import Node
 
 # prepend the timestamp in front of any log line
 logging.basicConfig(format="%(asctime)s %(message)s")
 
-SEED = int.from_bytes(os.urandom(8), byteorder="big")
-random.seed(SEED)
```

Comment:
Where's the random seed now?

Reply:
it's set in the localcluster, the same way